content/articles/wrap-up-2021.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "2021 CompilED Posts Wrap-up"
+date: 2021-12-31
+type: "post"
+authors: ["ctldev"]
+tags: ["wrap-up","reviews"]
+lede: "We look back at what we covered in CompilED for 2021, with new insights
+and accomplishments, and continuing challenges."
+shortlede: "We look back at what we covered in CompilED for 2021, with new
+insights and accomplishments, and continuing challenges."
+poster: "poster-wrap-up-2021.jpg"
+thumbnail: "thumbnail-wrap-up-2021.jpg"
+socmediaimg: "socmediaimg-wrap-up-2021.jpg"
+hiliteimg: "poster-wrap-up-2021.jpg"
+poster_sourceurl: "https://unsplash.com/photos/GrcMd1FvucY"
+poster_source: "Tai Bui on Unsplash"
+bookendanimal: "dragon"
+---
+
+We end the year by looking back at what we covered in CompilED for 2021. This
+year was still with its unique challenges of hybrid workspaces and learning
+environments, but we’re proud of our work in instructional technologies at the
+CTL.
+
+Our team develops both software and media for the teaching and learning
+community, so, we posted in CompilED three articles on the work in media
+production. We shared what we learned in developing many of our software
+projects, securing our infrastructure, and also touched upon the processes
+involved in user stories and experience. 
+
+And we’ll begin the new year year in resolution to continue our efforts in
```

Comment:
"we'll begin the new year year in resolution" - "we'll begin the new year with a resolution"